library/src/components-private/Error.vue

```diff
@@ -0,0 +1,86 @@
+<template lang="pug">
+  .container.pt-5.pb-10.error-page
```

Comment:
Let's use the class .nio-{component-name} at the base level of each component for consistency with other Tacklebox components.